optimism.tokenlist.json

```diff
@@ -707,6 +707,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 42,
+      "address": "0x002be8a5961e0f352092d6693133a6944b7846ba",
```

Comment:
Can you add my account as a minter to this test token `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` so I can test it as well? Alternatively pls issue some tokens to my account.

Reply:
sent, please check

Reply:
Git it thanks